app/controllers/v1/recipes_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class V1::RecipesController < ApplicationController
+  def create
+    recipe = Recipe.create(recipe_params)
+    if recipe.persisted?
+      render json: { message: 'The recipe was successfully created.' }
```

Comment:
I thought you always had to ad the status code 200 to a positive response when you are asserting a 200 status in a request spec?

Reply:
We changed that to 201 now, but generally, a response is always 200 unless you say anything else